jetty-ee9/pom.xml

```diff
@@ -309,485 +309,4 @@
-                      </rule>
-                      <rule>
-                        <!-- for ee9, we need to stay on 3.0.x of jaxws implementation -->
-                        <groupId>com.sun.xml.ws</groupId>
-                        <artifactId>jaxws-rt</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!3.0.).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- for ee9, we need to stay on 3.0.x of xml.ws api -->
-                        <groupId>jakarta.xml.ws</groupId>
-                        <artifactId>jakarta.xml.ws-api</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!3.0.).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- for ee9, we need to stay on 10.0.x of mortbay apache-jsp impl -->
-                        <groupId>org.mortbay.jasper</groupId>
-                        <artifactId>mortbay-apache-jsp</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!10.0.).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- for ee9, we need to stay on 10.0.x of mortbay apache-el impl -->
-                        <groupId>org.mortbay.jasper</groupId>
-                        <artifactId>mortbay-apache-el</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!10.0.).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- ignore unstable mina releases -->
-                        <groupId>org.apache.mina</groupId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>.+-M[0-9]+$</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                    </rules>
-                  </ruleSet>
-                </configuration>
-              </execution>
-            </executions>
-          </plugin>
-        </plugins>
-      </build>
-    </profile>
-  </profiles>
```

Comment:
Lets hold off on cleaning up the pom right now.
We can remove this section later, once we've confirmed the new dependabot configs.